scripts/trino-acl

```diff
@@ -49,8 +50,20 @@ def create_in_acl(acl, type, name, user_type, resource_name, allow, privileges):
     return acl
 
 
-def update_acl(acl, type, name, user_type, resource_name, allow):
-    print('This still needs to be implemented')
+def update_acl(acl, type, search_field, search_field_value, update_field, update_field_value, update_field_replacement):
+    try:
+        acl_type = acl[type]
+        acl_entry = [d for d in acl_type if d.get(search_field, "") == search_field_value][0]
+        user_group_list = acl_entry[update_field].split("|")
+        if(update_field_value):
+            user_group_list.remove(update_field_value)
+        user_group_list.append(update_field_replacement)
+        user_group_list_str = "|".join(user_group_list)
+        acl_entry[update_field] = user_group_list_str
+        acl_type_updated = [d if d.get(search_field, "") != search_field_value else acl_entry for d in acl_type] 
+        acl[type] = acl_type_updated
+    except:
```

Comment:
we should catch/handle specific Exceptions if we are expecting them here

Reply:
mmm, I'm not sure, this exception is expecting right values for accessing the json structure. We are capturing KeyError for the dictionary key access, and this won't create the output yml since the input values are wrong.